firmware/library/L1_Drivers/ST7066U.hpp

```diff
@@ -0,0 +1,196 @@
+
+// The file contains the driver class for the ST7066U Parallel LCD Driver.
```

Comment:
This usage section is a bit confusing. I would say remove it for now, and add it back in the next commit.